Wrapping/Python/Python.i

```diff
@@ -555,18 +554,15 @@
               # run the slice filter
               img = Slice(self, start=start, stop=stop, step=step)
 
-              if (slice_dim != -1):
-
-                # the stop is on the wrong side of step
-                if any( (s[1]-s[0])//s[2] <= 0 for s in sidx ):
-                  raise IndexError("invalid range")
-
+              if slice_dims:
                 size = img.GetSize();
 
                 # set the slice dimension size to 0
-                size = size[:slice_dim]+(0,)+size[slice_dim+1:]
+                for i in slice_dims:
+                  size = tuple(size[:i]) + (0,) + tuple(size[i+1:])
 
-                # reduce the 3D image to a 2D
+                print("extract size: ", size)
+                # Redfuce the dimension of the image
```

Comment:
'Reduce'

Reply:
That print statement shouldn't be their either.